### ISSUE TITLE
[Visual Requirements - Link]: Selected/Unselected state of ‘Lock time range and Shorten URL’ toggle buttons are not visible in high contrast mode.

### ISSUE THREAD

**Issue by natellium** [grafana/grafana#66308]
### User Experience:
Users with low vision who depend on this functionality will not be able to identify the state of control and interact with the controls properly and get disoriented if the selected/unselected state of buttons are not visible in High contrast mode.

### Pre-requisite: 
Windows - Applying High contrast theme. It doesn't matter what theme Grafana is using.

### Repro steps:
 Open #URL: https://play.grafana.org/  Sign in with valid credentials.  "Home" page appears. Navigate the page.  Navigate to the "Share Dashboard and panel" button and select it.  New pane opens with "Link" tab.  Navigate to the "Link"
tab. Apply high contract theme. Verify the issue.

### Actual Result:
Selected/unselected state of ‘Lock time range and Shorten URL’ toggle buttons are not visible in high contrast mode under 'Link tab'.  Expected Result
 Selected/unselected state of ‘Lock time range and Shorten URL’ toggle buttons should be visible in high contrast mode under 'Link tab'.  page. 

Note:  The same issue exists with the toggle buttons throughout the application. The same issue exists with "Raw query" checkbox under Explore screen. Selected radio button is not adapting the high contrast theme under 'Matching Instances' blade. Selected radio button is not adapting the high contrast theme under under 'View' tab.

### Severity
`SEV2`
